extensions/Worker.Extensions.CosmosDB/src/Worker.Extensions.CosmosDB.csproj

```diff
@@ -6,7 +6,7 @@
     <Description>Azure Cosmos DB extensions for .NET isolated functions</Description>
 
     <!--Version information-->
-    <VersionPrefix>4.8.1</VersionPrefix>
+    <VersionPrefix>5.0.0</VersionPrefix>
```

Comment:
Does this need a major version rev?

Reply:
Can we set connection mode to `Gateway` in `CosmosDBExtensionOptions`?

``` CSharp
CosmosClientOptions ClientOptions { get; set; } = new CosmosClientOptions() { ConnectionMode = ConnectionMode.Gateway };
```

Reply:
Yup, now with the new change of moving to CosmosDBExtensionOptions this is no longer a breaking change since we can now set a default